pylint/extensions/docparams.py

```diff
@@ -83,6 +77,22 @@ class DocstringParameterChecker(BaseChecker):
                   # we can't use the same old_name for two different warnings
                   # {'old_names': [('W9009', 'missing-yields-doc')]},
                  ),
+        'W9015': ('"%s" missing in parameter documentation',
+                  'missing-param-doc',
+                  'Please add parameter declarations for all parameters.',
+                  {'old_names': [('W9003', 'missing-param-doc')]}),
```

Comment:
I can't really do that, since registering old names with codes that already exist fails the pylint tests because the code that it tried to register as old-code for the new errors already exists..
You can check the failures [in this travis build](https://travis-ci.org/PyCQA/pylint/jobs/209237383).
I've reset back to my original suggestion..

Can you suggest a different implementation?